buildpipeline/pipeline.json

```diff
@@ -247,6 +247,23 @@
             "SubType": "uapaot"
           }
         },
+        {
+          "Name": "DotNet-CoreFx-Trusted-Windows",
+          "Parameters": {
+            "PB_Platform": "arm64",
+            "PB_BuildArguments": "-framework=uapaot -buildArch=arm64 -$(PB_ConfigurationGroup) -- /p:SignType=$(PB_SignType) /p:RuntimeOS=win10 /p:DotNetRestoreSources=$(PB_RestoreSource) /p:DotNetAssetRootUrl=$(PB_AssetRootUrl)",
+            "PB_BuildTestsArguments": "-framework=uapaot -buildArch=arm64 -$(PB_ConfigurationGroup) -SkipTests -Outerloop -- /p:RuntimeOS=win10 /p:ArchiveTests=true",
+            "PB_SyncArguments": "-p -- /p:ArchGroup=arm64 /p:RuntimeOS=win10 /p:TargetGroup=uapaot /p:DotNetRestoreSources=$(PB_RestoreSource) /p:DotNetAssetRootUrl=$(PB_AssetRootUrl)",
+            "PB_TargetQueue": "Windows.10.Arm64",
```

Comment:
@Chrisboh @MattGal is the Arm64 queue working or should I just disable test runs for this configuration for now?

Reply:
ARM64 machines are working and UWP F5 scenarios are running and passing there.  It's been a bit of a struggle to keep the machines from hanging during reboots recently and dumpling still needs to be configured there, otherwise stuff should work.

I'm confused though, why is the TargetQueue ARM64?  For uapaot, usually compilation to ARM is handled on AMD64 machines with ARM running on a SecondaryQueue.

Reply:
@weshaggard, I think we should just disable test runs for now for this new configuration. There is no matching compiler package available. Thank you!

Reply:
OK I will disable the tests for now.